pkg/rancher-components/src/components/Utils/DraggableZone/index.ts

```diff
@@ -0,0 +1 @@
+export { default as DraggableZone } from './DraggableZone.vue';
```

Comment:
don't know much about TS... Is this needed in order to import `DraggableZone` somewhere else?

Reply:
It's not mandatory.

You can leave that statement and import the component elsewhere with 
`import { DraggableZone } from '@components/Utils/DraggableZone';`

Or 

Remove it and import the component in this way:
`import DraggableZone from '@components/Utils/DraggableZone.vue';`

I opted for the first one because it seems to be used for all other components.